### DISCUSSION TITLE
URLs of camera not showed in Info sensor attributes

### DISCUSSION THREAD

**Discussion by alessandroarosio1977** [tsightler/ring-mqtt#638] [Support Q&A]
Hi all,
I have 2 StickUp Cameras and 1 Doorbell 4 installed.
I'm tring to find RTSP URL in Info sensor attributes but it is not showed.
Into that section, I found only:
BatteryLevel
BatteryLife
FirmwareStatus
LastUpdate
WirelessNetwork
WirelessSignal

Someone can help me?

**Comment by Greminn** [comment]
I have the same issue here... any ideas?

<img width="496" alt="image" src="https://github.com/tsightler/ring-mqtt/assets/81659316/c72beba2-ef21-46f5-8e83-d77d2bcb6f44">

**Comment by tsightler** [comment]
Currently the code depends on a call to device health actually working, if not, there is a case where these could be blank.  However, these are just informational, and the path is easy to figure out, it's just the device ID (show in logs during startup) + _live or _event.
